sdk/spring/azure-spring-data-cosmosdb/QueryPartitionedCollection.md

```diff
@@ -0,0 +1,58 @@
+### How to Query Partitioned Azure Cosmos DB Collection
+
+With Azure Cosmos DB, you can configure [partition key](https://docs.microsoft.com/en-us/azure/cosmos-db/partition-data) for your collection.
+
+Below is an example about how to query partitioned collection with this spring data module.
+
+#### Example 
+
+Given a document entity structure:
+```
+    @Document
+    @Data
+    @AllArgsConstructor
+    public class Address {
+        @Id
+        String postalCode;
+        String street;
+        @PartitionKey
+        String city;
+    }
+```
+
+Write the repository interface:
+```
+    @Repository
+    public interface AddressRepository extends DocumentDbRepository<Address, String> {
```

Comment:
There is no `DocumentDbRepository`. That is in old spring-data-cosmosdb SDK. 
This should extend `CosmosRepository` or `ReactiveCosmosRepository`